src/xdocs/config.xml

```diff
@@ -258,8 +258,8 @@
         Apart from the above mentioned properties, there are a few system
         properties that changes the way how Checkstyle behaves. For example,
         if you want to change the language of the output of Checkstyle, it
-        can be done by changing the property checkstyle.locale.language. All
-        the system properties can be found at the
+        can be done by changing the property<em>checkstyle.locale.language</em>.
```

Comment:
@romani This system property doesn't exist. This was removed in 4fd3722597bb03b76487110f24cb2985ccb4612e . Should we just remove this entire paragraph? The system properties page still does have the DTD load one.

Reply:
Let remove

Reply:
Alright! Thanks for mentioning this.

Reply:
@NitishKumar06 This is not done.

Reply:
I'll remove that